packages/core-components/src/components/ImageSlider.tsx

```diff
@@ -0,0 +1,58 @@
+import { Carousel } from '@material-tailwind/react';
+import { ThemeProvider } from '@material-tailwind/react';
+import React from 'react';
+
+import clsxm from '@/lib/clsxm';
+
+export type ImageSliderItemType = {
+  src: string;
+  name: string;
+  url: string;
+};
+
+type ImageSliderProps = {
+  imageSliderItems: ImageSliderItemType[];
+  imageSliderAutoPlay?: boolean;
+  imageSliderLoop?: boolean;
+  imageSliderAutoplayDelay?: number;
+  imageSliderHeight?: 'sm' | 'md' | 'lg';
+};
+
+const ImageSlider: React.FC<ImageSliderProps> = ({
+  imageSliderItems,
```

Comment:
if you notice, the prop naming here is a bit redundant. `items`, `autoPlay`, `loop`, `autoplayDelay` should be fine since it is already in the ImageSlider component context.

It's minor though